packages/vega-typings/types/runtime/scene.d.ts

```diff
@@ -84,9 +89,12 @@ export type SceneGroup = SceneItem & {
 
 export type SceneSymbol = SceneItem & {
   fill: string;
+  fillOpacity: number;
```

Comment:
A lot of these properties are optional in SceneRect. Are they optional here as well?

Reply:
I wasn't intending to make any assumptions about which fields should actually be optional yet. That being said, I'd assume `SceneRect` and `SceneSymbol` would match, regardless of optional or not.

Reply:
It's probably best to err on the side of optional since it's less strict.

Reply:
In this case, I would argue it would make sense then for `strokeWidth` to also become optional, since it would be strange for `stroke` and `strokeOpacity` to be optional, but not the width. It would also match with the changes proposed to `SceneRect`.